wait-for-it.sh

```diff
@@ -136,6 +141,17 @@ STRICT=${STRICT:-0}
 CHILD=${CHILD:-0}
 QUIET=${QUIET:-0}
 
+# check to see if timeout is from busybox?
+# check to see if timeout is from busybox?
+TIMEOUT_PATH=$(realpath $(which timeout))
+if [[ $TIMEOUT_PATH =~ "busybox" ]]; then
```

Comment:
You broke distros with NEW busybox: https://github.com/mirror/busybox/commit/c9720a761e88e83265b4d75808533cdfbc66075b

Happy debugging!

Reply:
Now it enters in the if condition and throws this 

```bash
wp_cli   | timeout: unrecognized option: t
wp_cli   | BusyBox v1.30.1 (2019-06-12 17:51:55 UTC) multi-call binary.
wp_cli   |
wp_cli   | Usage: timeout [-s SIG] SECS PROG ARGS
wp_cli   |
wp_cli   | Runs PROG. Sends SIG to it if it is not gone in SECS seconds.
wp_cli   | Default SIG: TERM.
```

Reply:
have anyone addressed this issue so far?

Reply:
I have fixed this by adding ```RUN apk update && apk add bash && apk add --no-cache coreutils``` to my dockerfile which updates timeout. Based on #71.